website/content/docs/release-notes/1.15.0.mdx

```diff
@@ -0,0 +1,241 @@
+      <th style={{verticalAlign: 'middle'}}>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+  
+  <tr>
+    <td rowSpan={2} style={{verticalAlign: 'middle'}}>
+      Vault Agent
+    </td>
+    <td style={{verticalAlign: 'middle', textAlign: 'center'}}>ENHANCED</td>
+    <td style={{verticalAlign: 'middle'}}>
+      Updated to use the latest Azure SDK version and Windows Identity
+      Federation (WIF).
+      <br /><br />
+      Learn more:&nbsp;
+      <a href="#">TBD</a>
+    </td>
+  </tr>
+  <tr>
+    <td style={{verticalAlign: 'middle', textAlign: 'center'}}>GA</td>
+    <td style={{verticalAlign: 'middle'}}>
+      Fetch secrets directly into your application as environment variables. 
+      <br /><br />
+      Learn more: <a href="/vault/docs/agent-and-proxy/agent/process-supervisor">Process Supervisor Mode</a>
+    </td>
+  </tr>
+
+  <tr>
+    <td style={{verticalAlign: 'middle'}}>
+      External plugins
+    </td>
+    <td style={{verticalAlign: 'middle', textAlign: 'center'}}>BETA</td>
+    <td style={{verticalAlign: 'middle'}}>
+      Run external plugins in their own container with native container platform
+      controls.
+      <br /><br />
+      Learn more: <a href="/vault/docs/plugins/containerized-plugins">Containerize Vault plugins</a>
+    </td>
+  </tr>
+
+  <tr>
+    <td style={{verticalAlign: 'middle'}}>
+      Eventing
+    </td>
+    <td style={{verticalAlign: 'middle', textAlign: 'center'}}>BETA</td>
+    <td style={{verticalAlign: 'middle'}}>
+      Subscribe to notifications for various events in Vault. Includes support
+      for filtering, permissions, and cluster configurations with K-V secrets.
+      <br /><br />
+      Learn more: <a href="/vault/docs/concepts/events">Events</a>
+    </td>
+  </tr>
+
+  <tr>
+    <td style={{verticalAlign: 'middle'}}>
+      Auditing
+    </td>
+    <td style={{verticalAlign: 'middle', textAlign: 'center'}}>ENHANCED</td>
+    <td style={{verticalAlign: 'middle'}}>
+      Upgrades and improves the audit logging backend.
+    </td>
+  </tr>
+
+  <tr>
+    <td rowSpan={2} style={{verticalAlign: 'middle'}}>
+    Vault GUI
+    </td>
+    <td style={{verticalAlign: 'middle', textAlign: 'center'}}>GA</td>
+    <td style={{verticalAlign: 'middle'}}>
+      New LDAP secrets engine GUI.
+      <br /><br />
+      Learn more: <a href="/vault/docs/configuration/ui">Vault UI guide</a>
+    </td>
+  </tr>
+  <tr>
+    <td style={{verticalAlign: 'middle', textAlign: 'center'}}>ENHANCED</td>
+    <td style={{verticalAlign: 'middle'}}>
+      &bull; Improved backend LDAP management and landing page dashboard.<br />
+      &bull; View secrets you have read access to under your directory.<br />
+      &bull; View diffs between previous and new secret versions.<br />
+      &bull; Copy and paste secret paths from the GUI to the Vault CLI or API.
+      <br /><br />
+      Learn more: <a href="/vault/docs/configuration/ui">Vault UI guide</a>
+    </td>
+  </tr>
+
+  <tr>
```

Comment:
Replication should probably go in the Enterprise section.

Reply:
Moved